src/runtime/subscription-linking-flow.ts

```diff
@@ -169,13 +169,78 @@ export class SubscriptionLinkingFlow {
       throw new Error('Missing required field: publisherProvidedId');
     }
     const publicationId = this.pageConfig_.getPublicationId();
-    return this.startMultipleLinks({
-      linkTo: [{publicationId, publisherProvidedId}],
-    }).then((result) => {
-      return {
-        success: !result.anyFailure,
-        publisherProvidedId: result.links[0].publisherProvidedId,
-      };
+    const args = feArgs({publicationId});
+    const activityIframeView = new ActivityIframeView(
+      this.win_,
+      this.activityPorts_,
+      feUrl('/linksaveiframe', {
+        subscriptionLinking: 'true',
+        ppid: publisherProvidedId,
+      }),
+      args,
+      /* shouldFadeBody= */ false
+    );
+
+    activityIframeView.onCancel(() => {
```

Comment:
Optional: what do you think of splitting these handlers into their own methods?

Reply:
I think I am going to delete these in fairly short order when we finish migrating to the new UI and that it's not really necessary for any reason.  Perhaps the ones in startMultipleLinks should get migrated to their own functions when we clean that up.